### PR TITLE
Update OpenCover and ReportGenerator versions

### PR DESCRIPTION
**Pull request by stephentoub**
Our code coverage runs recently started failing consistently in CI, and it looks like there may be some related fixes in more recent releases.

cc: @mmitche, @weshaggard

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
       "Microsoft.NETCore.Console": "1.0.0-rc2-23530",
 
       "coveralls.io": "1.4",
-      "OpenCover": "4.6.166",
-      "ReportGenerator": "2.3.4",
+      "OpenCover": "4.6.261-rc",
```
Comment (weshaggard) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json]: You will need to update the related versions in corefx repo as well once this gets integrated.

Comment (stephentoub) [inline src/Microsoft.DotNet.Build.Tasks/PackageFiles/test-runtime/project.json]: Yup, in the test-runtime stuff, right?  Already have that done locally.

**Comment by weshaggard** [comment]
LGTM. You will need to wait to update the BuildTools package for corefx however, as we are in the middle of a transition that is breaking that we need to consume before you can update with this change.

FYI @joperezr

**Comment by stephentoub** [comment]
Ok, thanks.  When do we expect that to be completed?

**Comment by joperezr** [comment]
We are currently working on it trying to get a successful build. I'll keep you posted.

**Comment by stephentoub** [comment]
Thanks, @joperezr.

**Pull request merged**